src/ros_tcp_endpoint/default_server_endpoint.py

```diff
@@ -19,7 +19,7 @@ def main():
 
     # Start the Server Endpoint
     rospy.init_node(ros_node_name, anonymous=True)
-    tcp_server.start()
+    tcp_server.start({})
```

Comment:
What's the motivation behind calling start with an empty dict? Wouldn't the dict already initialize to empty when the server is constructed?

Reply:
Yeah, this can be removed.